pkg/backend/filestate/backend.go

```diff
@@ -525,7 +539,7 @@ func (b *localBackend) apply(
 		} else {
 			link, err = b.bucket.SignedURL(context.TODO(), b.stackPath(stackName), nil)
 			if err != nil {
-				return changes, result.FromError(errors.Wrap(err, "Could not get signed url for stack location"))
+				cmdutil.Diag().Warningf(diag.Message("", "Could not get signed url for stack location: %v"), err)
```

Comment:
1. why did this change?
2. Why have this be a warning?

Reply:
answered in slack.  but can you add comments explaining what the issue is.

Reply:
can we only warn in the gcs case?

Reply:
I think it's important to warn in all cases rather than fail because other clouds may have their own nuances with credentials and signed urls, although I haven't done my research here to confirm. In the case that there's a real error getting the perma link I think it's still not a good reason to fail a pulumi run. If all the infrastructure was updated successfully it would be misleading for users to see a failed CI job for a failed permalink.

Reply:
Thought about this a bunch, and that makes sense to me as well.  Thanks!